extras/ds18b20/ds18b20.h

```diff
@@ -0,0 +1,20 @@
+#ifndef DRIVER_DS18B20_H_
+#define DRIVER_DS18B20_H_
+
+typedef struct {
+	uint8_t id;
+    uint8_t major;
+    uint8_t minor;
+} ds_sensor_t;
+
+// Scan all ds18b20 sensors on bus and return its amount.
+// Result are saved in array of ds_sensor_t structure.
+// Cause printf in esp sdk don`t support float,
+// I split result as two number (major, minor).
```

Comment:
I hadn't noticed this comment before and had been wondering about this..

I've opened bug #95 for the printf issue, so hopefully we can get that fixed..  In the meantime, though, this really isn't the right way to work around it anyway.  We should really just return a float from these routines, and then the caller can do a conversion *if* they want to print it..  This way makes it easier to print but substantially harder to use for any other purpose.